extensions/positron-r/package.json

```diff
@@ -667,6 +667,11 @@
         "view": "testing",
         "contents": "No testthat tests found.\n[Add a test](command:r.useTest)",
         "when": "isRPackage && config.positron.r.testing && testthatIsConfigured && !testthatHasTests"
+      },
+      {
+        "view": "debug",
+        "contents": "Positron currently provides initial debugging support for R code.\n[Learn more](https://positron.posit.co/guide-r.html#debugging)",
+        "when": "debugStartLanguage == 'r'"
```

Comment:
Nice! We'll be able to use this to provide a "Pause" button to interrupt a running R command and drop in the debugger. This would be ideal with a context key that indicates whether the current console is busy.